### DISCUSSION TITLE
Does MSQUIC implement any sort of forward error correction features?

### DISCUSSION THREAD

**Discussion by cjgatto** [microsoft/msquic#3517] [Q&A] [answered]
As far as I can tell, QUIC RFC doesn't show any support for FEC, but I'm unsure if it is present as another name or methodology. 

Does MSQUIC support any sort of mechanism for proactively reducing retransmission cost & delay?

**Comment by nibanks** [comment, marked as answer]
No, there is no FEC in the QUIC protocol, nor MsQuic.
